tox.ini

```diff
@@ -1,9 +1,6 @@
 [tox]
 envlist = py{3.8,3.9,3.10,3.11,3.12}-{lint,test}-{basic,yaml,hcl}
 skip_missing_interpreters = true
-# Prevent Tox from ignoring `deps` and installing the package (without
-# development dependencies) in the environment
-skipsdist = true
```

Comment:
@frank-lenormand We don't need the app installed for the linting but we do for the cli tests cases.

Reply:
I added this line because Tox was ignoring the requirements files entirely, and installing the package itself in the virtual environment directly.

`mypy` is pinned in `dev-requirements.txt`, and this option is only defined in the `[tox]` section.